### PR TITLE
Removing Translations

### PR DESCRIPTION
**Pull request by cpholguera**
As agreed, removed existing translations and included an explanatory note to the readme.

### CODE REVIEW

**Comment by OMTGreleaser** [comment]
Results for commit 39c11d6427f77186d16f8aa2be647c4f5300862f: Broken link result: [!] Error(s) found by the Links validator: 10 pages have dead links! Verbose output in /link-check-result.out .
 Please run the Apply_Link_Check.sh in the tools folder to see wherhe the Link issues are. 
 Markdown result errorlength:       11 ../linter-result.out .
 Please run the Apply_Linter_check.sh in the tools folder to see where the Lint issues are.

File: README.md [hunk 1]
```diff
@@ -93,25 +93,17 @@ The release 1.0 of the MSTG was published in June 2018. You can get intermediate
...
-- [Xamarin Compliancy Checklist (WIP)](https://drive.google.com/open?id=1UL1yLRREJwXfe0HlrcX-IuvPYQM7lTtG "Xamarin Compliancy Checklist").
-
-If there's something you really want to see in the guide, or you want to suggest an improvement, create an issue [issue](https://github.com/OWASP/owasp-mstg/issues "Issue") or ping us on [Slack](https://owasp.slack.com/messages/project-mobile_omtg/details/ "Come to our Slack!").
-If the issue is accepted, we will schedule it into one of our [milestones](https://github.com/OWASP/owasp-mstg/milestones "Milestones").
+> The MSTG is a living document that changes and adapts to the most recent security recommendations every day. While we do want to reach the maximum audience possible, our past experience shows that maintaining translations has proven to be an extremely challenging task. You're free to do forks for your own translations but please understand that any PRs containing them will be declined due to the previously mentioned reasons. However, we strongly encourage further translations of the MASVS as it is much easy to maintain.
```
Comment (sushi2k) [inline README.md:106]:
```suggestion
> The MSTG is a living document that changes and adapts to the most recent security recommendations every day. While we do want to reach the maximum audience possible, our past experience shows that maintaining translations has proven to be an extremely challenging task. You're free to do forks for your own translations but please understand that any PRs containing them will be declined due to the previously mentioned reasons. However, we strongly encourage further translations of the MASVS as it is much easier to maintain.
```

File: README.md [hunk 2]
```diff
@@ -93,25 +93,17 @@ The release 1.0 of the MSTG was published in June 2018. You can get intermediate
...
+**We are searching for additional authors, reviewers and editors.**
 
-Drop a us line on the [Slack channel](https://owasp.slack.com/messages/project-mobile_omtg/details/ "Come to our Slack!") before you start working on a topic. This helps us to keep track of what everyone is doing and prevent conflicts. You can create a Slack account here:
+You can start contributing on many different ways ([filing or reporting issues](https://github.com/OWASP/owasp-mstg/issues "our issues section"), browsing the [existing content](https://mobile-security.gitbook.io/mobile-security-testing-guide/ "existing content") and proofread it, etc.), please check our [Contribution Guide](https://github.com/OWASP/owasp-mstg/blob/master/CONTRIBUTING.md "Contribution Guide") for ideas and before you get started.
```
Comment (sushi2k) [inline README.md:100]:
```suggestion
You can start contributing on many different ways:

- [filing or reporting issues](https://github.com/OWASP/owasp-mstg/issues/new/choose "New Issue"),
- [browsing through existing issues and work on them](https://github.com/OWASP/owasp-mstg/issues "our issues section") or
- browsing the [existing content](https://mobile-security.gitbook.io/mobile-security-testing-guide/ "existing content") and proofread it.

Please check our [Contribution Guide](https://github.com/OWASP/owasp-mstg/blob/master/CONTRIBUTING.md "Contribution Guide") for ideas and before you get started.
```

**Comment by OMTGreleaser** [comment]
Results for commit 07978ebebb9af85625c77d1519c13ccafc17ac23: Broken link result: [!] Error(s) found by the Links validator: 23 pages have dead links! Verbose output in /link-check-result.out .
 Please run the Apply_Link_Check.sh in the tools folder to see wherhe the Link issues are. 
 Markdown result errorlength:       46 ../linter-result.out .
 Please run the Apply_Linter_check.sh in the tools folder to see where the Lint issues are.

**Comment by OMTGreleaser** [comment]
Results for commit 087b244f01dc6c46f9aea47e2bdf5be5e23ec874: Broken link result: [!] Error(s) found by the Links validator: 22 pages have dead links! Verbose output in /link-check-result.out .
 Please run the Apply_Link_Check.sh in the tools folder to see wherhe the Link issues are. 
 Markdown result errorlength:       46 ../linter-result.out .
 Please run the Apply_Linter_check.sh in the tools folder to see where the Lint issues are.

**Pull request merged**